src/commands/wrap.ts

```diff
@@ -1,18 +1,20 @@
 import { Argv } from "yargs";
 import { wrap } from "../implementations/wrap";
 import { transformValidationErrors } from "../implementations/wrap/ajvErrorTransformer";
+import { isDir } from "../implementations/wrap/diskUtils";
 
 import signale from "signale";
 
 interface WrapCommand {
   rawDocumentsPath: string;
-  wrappedDocumentsDir: string;
   schema: any;
   openAttestationV3: boolean;
   unwrap: boolean;
+  outputFile: any;
```

Comment:
why any ? looks like it's `outputFile?: string;`

Reply:
i followed the typing of schema which is a string too

Reply:
change schema :)